common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -120,49 +159,199 @@ hfe_initializer_egypt_main = {
 							show_notification = no
 						}
 						create_leader = {
-							name = Amonhirkhopshef
+							name = "Amonhirkhopshef Mutatistep"
 							species = event_target:hfe_egypt_species
 							class = ruler
-							skill = 5
+							skill = 10
 							leader_age_min = 250
 							leader_age_max = 650
+							immortal = yes
+							gender = male
```

Comment:
intended to have random gender

Reply:
Dude, you set it so that if the Ju Ahet's ruler is randomly spawned as a woman, THAT LEADER WOULD BE KILLED AND A MAN WOULD BE CREATED. You always talk about unnecessary code and I don't care, but here is one of the weirdest ones!

Just have it generate a man instead of killing a woman if she spawns and spawning a man _anyway_!